weather-app/src/components/InfoBox.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Text, View, StyleSheet } from "react-native";
+import { Feather } from "@expo/vector-icons";
+
+const InfoBox = (props) => {
+  const { wind, humidity, chanceOfRain } = props;
+  return (
+    <View
+      style={[
+        styles.flexRow,
+        {
+          backgroundColor: "rgba(34, 107, 163, 0.7)",
+          borderRadius: 30,
+          borderColor: "white",
+          borderWidth: 1,
+        },
+      ]}
+    >
+      <View style={{ flex: 1 }}>
+        <View style={styles.flexRow}>
+          <Feather name="wind" size={20} color="white" />
+        </View>
+        <View style={styles.flexRow}>
+          <Text style={{ color: "white", fontSize: 17 }}>{wind}</Text>
+        </View>
```

Comment:
Whenever you have repeating sections of code in React and React Native, extract them into seperate components